packages/template-retail-react-app/app/pages/checkout/partials/payment.jsx

```diff
@@ -130,7 +226,7 @@ const Payment = () => {
 
                     <Box pt={3}>
                         <Container variant="form">
-                            <Button w="full" onClick={reviewOrder}>
+                            <Button w="full" onClick={onSubmit}>
```

Comment:
How the current behaviour differs from the [original](https://pwa-kit.mobify-storefront.com/global/en-GB/checkout):
- When not filling out the form, clicking on Review Order button no longer scrolls you up to reveal the first error.
- Form validation: entering a correct credit card number does not immediately clear its error message.

Reply:
fixed